### PR TITLE
Add tricks

### PR DESCRIPTION
**Pull request by berk245**
## Description

This PR adds creating new tricks feature. In the gallery, we render a an add new button that opens a modal with a form. After a successful submission, the tricks list get updated automatically. 

## Task
[Notion Task Link](https://www.notion.so/Tasks-538c1c6dbb2f4cba9a11c204ff418786?p=90a388d14eb0437bb47a5975df23ab23&pm=s)

## Screenshots (optional)
image
image

### CODE REVIEW

File: src/lib/handlers/db.js [hunk 1]
```diff
@@ -42,26 +47,37 @@ const createUserDocument = async (user) => {
 
 export const deleteTrick = async (trickId) => {
 	try {
-		const isConfirm = confirm('Are you sure?')
-		if(!isConfirm) return
-		setLoading(true)
+		const isConfirm = confirm('Are you sure?');
+		if (!isConfirm) return;
+		setLoading(true);
 		const userRef = doc(db, 'users', user.uid);
```
Comment (tonyjojo) [inline src/lib/handlers/db.js:53]: I would make these (`'users', 'tricks'`) constants, in a separate `constants` file, so if it ever changes it's more constistent

File: src/lib/handlers/db.js [hunk 2]
```diff
@@ -42,26 +47,37 @@ const createUserDocument = async (user) => {
...
-			};
-		});
-		setLoading(false)
+		getUpdatedTricks(user);
 	}
 };
 
+export const createNewTrick = async (user, trick) => {
+	const userRef = await getUserRef(user);
+	const tricksCollection = collection(userRef, 'tricks');
```
Comment (tonyjojo) [inline src/lib/handlers/db.js:67]: I would make these (`'users', 'tricks'`) constants, in a separate `constants` file, so if it ever changes it's more constistent

**Review by tonyjojo — APPROVED** [review]
Looking good, just a small comment about constant values in the db handler

File: src/lib/handlers/db.js (hunk 2 above)
Comment (berk245) [reply src/lib/handlers/db.js:67]: I am not 100% sure what this means but would be happy to hear more about it

**Review by niccoloMG — APPROVED** [review]
lgtm. thanks for the PR

File: src/lib/handlers/db.js (hunk 2 above)
Comment (berk245) [reply src/lib/handlers/db.js:67]: I am merging this now. Let's talk about possible optimizations later

**Pull request merged**